docs/tools/install.sh

```diff
@@ -112,6 +112,17 @@ function check_directcsi_consistency() {
     fi
 }
 
+function is_greater_than_or_equal_to_4.1.0() {
```

Comment:
You could add the support like below
```diff
diff --git a/docs/tools/install.sh b/docs/tools/install.sh
index 8d38bdb1..2ebcdb4f 100755
--- a/docs/tools/install.sh
+++ b/docs/tools/install.sh
@@ -28,6 +28,10 @@ function run_plugin_cmd() {
     "${plugin_cmd[@]}" "$@"
 }
 
+function is_psp_found() {
+    kubectl get --ignore-not-found=true psp directpv-min-io --no-headers -o NAME | grep -q .
+}
+
 function is_legacy_drive_found() {
     if ! kubectl get --ignore-not-found=true crd directcsidrives.direct.csi.min.io --no-headers -o NAME | grep -q .; then
         return 1
@@ -180,6 +184,13 @@ function wait_for_crd() {
 }
 
 function main() {
+    if is_psp_found; then
+        echo "[WARNING] Older PodSecurityPolicy found !!!"
+        echo "[WARNING] DirectPV removed PodSecurityPolicy support as per Kubernetes supportability"
+        echo "[WARNING] This installation does not change anything related to existing PodSecurityPolicy"
+        echo "[WARNING] You have to remove it manually and use Pod Security Admission as per your requirement"
+    fi
+
     echo "* Probe legacy DirectCSI"
     legacy=0
     migrate=0
```

Reply:
Don't we need the directpv version check? sometimes users will just take the script from master and use.

Reply:
`master` always has the latest code and users should not do that for older releases.